### PR TITLE
Automatically retrieved `aws_account_id` from aws profile instead of prompting

### LINKED ISSUE #1714: [BUG]: `aws_account_id` is None when creating uber principal
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

I am getting an error on running `databricks labs ucx create-uber-principal` as the `aws_account_id` is being set as None. I see in the code that this value is fetched from `named_parameters`. How to pass the `aws_account_id` as a `named_parameter` via CLI?

### Expected Behavior

_No response_

### Steps To Reproduce

_No response_

### Cloud

AWS

### Operating System

macOS

### Version

latest via Databricks CLI

### Relevant log output

_No response_

### PR DESCRIPTION
**Pull request by nkvuong**
## Changes
- Automatically retrieved `aws_account_id` from aws profile instead of prompting users for this.
- Documented the optional kms-key flag for create roles
cc: @mohanab-db 

### Linked issues

Resolves #1714

### Tests

- [x] manually tested
- [x] verified on staging environment (screenshot attached)

### CODE REVIEW

File: labs.yml [hunk 1]
```diff
@@ -128,6 +128,8 @@ commands:
     flags:
       - name: aws-profile
         description: AWS Profile to use for authentication
+      - name: kms-key
+        description: (Optional) KMS Key to be specified for the UC roles
       - name: single-role
```
Comment (FastLee) [inline labs.yml:133]: While you're at it. Add the role_name and policy_name parameters.

**Review by FastLee — CHANGES_REQUESTED** [review]

File: labs.yml (hunk 1 above)
Comment (nkvuong) [reply labs.yml:137]: added

**Review by FastLee — APPROVED** [review]

File: src/databricks/labs/ucx/cli.py [hunk 2]
```diff
@@ -303,7 +305,7 @@ def create_missing_principals(
     if not ctx:
         ctx = WorkspaceContext(w, named_parameters)
     if ctx.is_aws:
-        return ctx.iam_role_creation.run(prompts, single_role=single_role)
+        return ctx.iam_role_creation.run(prompts, single_role=single_role, role_name=role_name, policy_name=policy_name)
```
Comment (FastLee) [inline src/databricks/labs/ucx/cli.py:308]: For some reason, single_role defaults to False.

**Review by FastLee — CHANGES_REQUESTED** [review]

**Review by nfx — APPROVED** [review]
lgtm

**Pull request merged**